static/scripts/main.js

```diff
@@ -6,23 +6,39 @@ window.onload = function () {
     if(hash) {
       hash.scrollIntoView()
     }
+    $('[data-md-sticky-header]').headroom({
+      offset: UnpinOffset,
+      tolerance: 0
+    });
 }
 
-var CurrentScroll = 0;
+var SearchbarTop = 61;
```

Comment:
What does this magic number stands for? The height of the header? Can it be computed dynamically?